cl/audio/tests.py

```diff
@@ -48,6 +67,36 @@ def test_do_jurisdiction_podcasts_have_good_content(self) -> None:
             ("//channel/item/title", 2),
             ("//channel/item/enclosure/@url", 2),
         )
+        print("XML T")
```

Comment:
I think We can remove this print statement

Reply:
Sure, I missed it. Removed.